### ISSUE TITLE
parameter mis-match for AWS

### ISSUE THREAD

**Issue by 7hunderbird** [genesis-community/bosh-genesis-kit#37]
When trying to generate a proto-BOSH for AWS, this error occurred:

```
./hooks/new: line 247: aws_key_name: unbound variable
Could not create new env us-west-test (in /Users/tbird/code/sandbox/genesis/bosh-deployments): 'new' hook exited 1
```
The parameter being set at the beginning of ./hooks/new is called `aws_access_key`, yet it's looking for `aws_key_name`.

**Comment by jhunt** [comment]
what version of the kit are you running?  I can't find references to aws_key_name in anywhere in `hooks/new`

**Comment by jhunt** [comment]
`aws_key_name` and `aws_access_key` are two very different things.  This is fallout from the `vcap@` stuff @wayneeseguin and I hacked in a few months back.  Fix incoming!

**Comment by 7hunderbird** [comment]
I'm just being a newbie.  

In the manual it explains how I needed to add the name of the key-pair, just the access_key and secret.  

https://github.com/genesis-community/bosh-genesis-kit/blob/master/MANUAL.md#deploying-to-amazon-web-services

**Comment by jhunt** [comment]
Did you create a keypair named `vcap@us-west-test`, like the new script recommended?

**Comment by jhunt** [comment]
Fix for this will be going out in 1.1.2 of the kit, once it finishes its trial-by-pipeline.

**Comment by jhunt** [comment]
https://github.com/genesis-community/bosh-genesis-kit/releases/tag/v1.1.1
